Cargo.toml

```diff
@@ -88,8 +88,10 @@ tracing-subscriber = "0.2"
 url = "2.1"
 
 [dev-dependencies]
+bigdecimal = { version = ">= 0.0.10, < 0.2.0" }
```

Comment:
unfortunately `diesel` does not reexport the version that they depend on, so we have to add this here ourselves